### ISSUE TITLE
Source data type "Pimcore report" is not working

### ISSUE THREAD

**Issue by studio1gmbh-pimcore** [BlackbitDigitalCommerce/pimcore-data-director#155]
### Data director version
3.6.1 (also in 3.5.36, 3.5.43, 3.5.3 3.5.54)

### Steps to reproduce
1. Add new dataport
2. Select "Pimcore Report" as source data type

### Actual Behavior
Data director GUI is crashing. The DevTools' console of the browser shows following javascript error:

> script-proxy?storage…140da582db23a:74382 Uncaught TypeError: Cannot read properties of undefined (reading 'groups')
    at constructor.renderReportSourceConfigPanel (script-proxy?storage…da582db23a:74382:51)
    at constructor.renderSourceConfigPanel (script-proxy?storage…da582db23a:68504:22)
    at constructor.<anonymous> (script-proxy?storage…da582db23a:68434:34)
    at constructor.fire (ext-all.js?_dc=98f77…0da582db23a:2407:19)
    at constructor.doFireEvent (ext-all.js?_dc=98f77…da582db23a:2464:104)
    at constructor.doFireEvent (ext-all.js?_dc=98f77…da582db23a:4851:110)
    at prototype.doFireEvent (ext-all.js?_dc=98f77…da582db23a:4452:267)
    at constructor.fireEventArgs (ext-all.js?_dc=98f77…da582db23a:2457:223)
    at constructor.fireEvent (ext-all.js?_dc=98f77…da582db23a:2455:525)
    at constructor.checkChange (ext-all.js?_dc=98f77…a582db23a:10072:444)

This error is pointing to `Resources/public/js/components/DataportPanel.js` line 6405.
```js
        for (var i = 0; i < pimcore.report.broker.groups.length; i++) {
```

Pimcore backend is not usable until the browser tab is completely reloaded.

### Expected Behavior
Data director imports with source "Pimcore Report" are possible and no error occurs.

**Comment by Alanin** [comment]
Created a fix and a pull request https://bitbucket.org/blackbitwerbung/pimcore-plugins-data-director/pull-requests/141

**Comment by Alanin** [comment]
Fixed in 3.6.15
